api/Service/CategoryService.cs

```diff
@@ -0,0 +1,91 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using YonderfulApi.Models;
+using YonderfulApi.Data;
+using Microsoft.EntityFrameworkCore;
+using YonderfulApi.DTOs;
+using Microsoft.AspNetCore.Components;
+
+namespace YonderfulApi.Service
+{
+    public class CategoryService : ICategoryService
+    {
+        private readonly DataContext _context;
+        private readonly IPictureService _pictureService;
+        public CategoryService(DataContext context, IPictureService pictureService) {
+            _context = context;
+            _pictureService = pictureService;
+        }
+
+    public async Task<Category> GetCategory(int categoryId) {
```

Comment:
Small thing here: alignment of the methods is off. Reformat the document (ALT + SHIFT + F)